### DISCUSSION TITLE
Opening previous analysis results

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/topobank#885] [Q&A] [answered]
How can I open previous analysis results, without having to recalculate them?

I click on 'Show Analyses' which brings me to this page 
![Screenshot 2022-08-08 124409](https://user-images.githubusercontent.com/108830413/183480608-184a7a0d-d381-428d-b182-728b4b724890.jpg)
but I do not find a list of my previous analyses here.

**Comment by mcrot** [comment, marked as answer]
Hey @sitangshugk95 ,

Thanks for your question. This information box means that you have not chosen any measurements for analysis yet.
Especially not those measurement from your previous analysis.
The system does not know for which measurements you want to see the contact analysis (aka: which are "selected").
There are two ways to select measurements for analysis: 

**Solution 1**: Switch to the *Find & Select* tab. Choose one or more digital surface twins and/or one more more single measurements.
There you can expand each digital surface twin and see the measurements which belong to that twin.
Each digital surface twin and each measurement has a checkbox on the left side which can be used.
After clicking the checkboxes, you'll find your selection underneath the navigation bar.
Then click on the *Analyze* button. Example:

![grafik](https://user-images.githubusercontent.com/9113541/183751037-f95cb751-4eb1-4ea2-a943-05c107aab4cc.png)

**Solution 2**: Switch to the detail view of the digital surface twin or measurement. Press the button *Analyze this digital surface twin* or
*Analyze this measurement*, respectively. This digital surface twin or measurement will then become the new selection and the old selection will be discarded. Example:
![grafik](https://user-images.githubusercontent.com/9113541/183751142-62739c6f-73d5-41ed-b1b2-4f379eb66182.png)

Concerning your "previous" analysis: When you select the same subjects for analysis as before for contact mechanics,
they will be shown with the same parameters as you have used before. Already calculated analyses are save and will be loaded again.

Does this help?
